tkltest/util/config_util.py

```diff
@@ -886,7 +872,7 @@ def resolve_classpath(tkltest_config, command):
                 os.remove(jar_file_path)
             else:
                 __collect_jar_modules(jar_file_path, jars_modules)
-    elif app_build_type == 'maven':
+    elif app_build_type in ['maven', 'gradle']:
         for jar_file_path in class_path_order:
             if jar_file_path.endswith('.jar'):
                 __collect_jar_modules(jar_file_path, jars_modules)
```

Comment:
'__collect_jar_modules' -> the name of this module is misleading, because it is not actually collecting modules. Can you change it to a more reflective name?

Reply:
OK